user-guide/Advanced_Functionality/Security/DataMiner_user_permissions.md

```diff
@@ -412,7 +412,10 @@ Available from DataMiner 10.2.0/10.1.10 onwards. Permission to send emails via t
 
 ### Modules \> API
 
-These user permissions are used for the API deployment feature, which is currently still in [soft launch](xref:SoftLaunchOptions).
+These user permissions are used for the API deployment feature, which is in [soft launch](xref:SoftLaunchOptions).
+
+>[!NOTE]
+> This feature has been replaced by User-Defined APIs and will be removed in the future.
```

Comment:
```suggestion
> This feature has been marked obsolete from DataMiner version 10.3.6. It will be replaced by User-Defined APIs and will be removed in the future.
```